lib/schema.js

```diff
@@ -446,6 +446,26 @@ module.exports = {
           }
         },
         {
+          "description": "Paginate over list members in the given mailing list",
+          "href": "/lists/{address}/members/pages",
+          "method": "GET",
+          "title": "page",
```

Comment:
I agree, only problem is the response between `GET /list/{address}/members` and `GET /list/{address}/members/pages`  is different. 

by the way, when I use `GET /lists/{address}/members`, `body.total_count`  is always return `10000` (I have 10000+ members in my mailing list), maybe it's a `mailgun` bug